### DISCUSSION TITLE
Possible to span the tabs across the screen?

### DISCUSSION THREAD

**Discussion by EugeneKallis** [carbon-design-system/carbon-components-svelte#1196] [Q&A] [answered]
Just trying out this comp lib and wondering if its possible to span the tabs across the screen. They seem to just stay toward the left. Im using tailwindcss along with this and try setting it in flex box and centering but wouldnt work.

**Comment by metonym** [comment, marked as answer]
Released in [v0.103.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.103.0)

Example: https://svelte.carbondesignsystem.com/components/Tabs#full-width
